spec/controllers/recommendations_controller_spec.rb

```diff
@@ -0,0 +1,78 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe RecommendationsController, type: :controller do
```

Comment:
We need to test that if a user presses 'Save and come back later' on the review form and the recommendation doesn't save the form is re-rendered with errors.

However currently there are no real errors that can be triggered in this scenario!

So adding a couple of controller specs, including one where I stub the recommendation to fail to save.

Hope this makes sense!